scripts/sync-org-repositories.py

```diff
@@ -0,0 +1,518 @@
+                else:
+                    if sync_file(source_path, dest_path, dest_rel_path):
+                        files_changed.append(dest_rel_path)
+
+            if not files_changed:
+                print(f"All files up to date for {repo_name}")
+                return True
+
+            if dry_run:
+                print(f"[DRY RUN] Would create PR with {len(files_changed)} file(s)")
+                return True
+
+            # Step 4: Create branch, commit, and push to fork
+            branch_name = f"sync-repo-standards-{datetime.now().strftime('%Y%m%d%H%M%S')}"
+            commit_message = "chore: sync repository standards\n\nUpdated files:\n" + \
+                           "\n".join(f"- {f}" for f in files_changed)
+
+            print("\nCreating branch and committing changes...")
+            if not create_branch_and_commit(repo_path, branch_name, files_changed, commit_message):
+                return False
+
+            # Step 5: Create pull request from fork to upstream
+            pr_title = "chore: sync repository standards"
+            pr_body = f"""This PR synchronizes repository standards from org-infra.
+
+## Files Updated
+{chr(10).join(f"- `{f}`" for f in files_changed)}
+
+## Description
+This is an automated PR to ensure repository settings are consistent across the organization.
+
+---
+*This PR was automatically generated by the sync_org_repositories workflow.*
+"""
+
+            print("Creating pull request from fork to upstream...")
+            return create_pull_request(org, repo_name, fork_owner, branch_name, pr_title, pr_body)
+        except subprocess.CalledProcessError as e:
+            print(f"Error processing {repo_name}: {e}")
+            return False
+        except Exception as e:
+            print(f"Unexpected error processing {repo_name}: {e}")
+            import traceback
+            traceback.print_exc()
+            return False
+
+
+def main():
+    args = parse_args()
+
+    if not GITHUB_TOKEN:
+        print("Error: GITHUB_TOKEN or GITHUB_PAT environment variable not set")
+        sys.exit(1)
+
+    # Get the authenticated user/app name
+    fork_owner = get_authenticated_user()
+    if not fork_owner:
+        print("Error: Could not determine authenticated user/app")
+        sys.exit(1)
+
+    print(f"Authenticated as: {fork_owner}")
+    config = load_sync_config(args.config)
+
+    # Fetch and parse peribolos.yml
+    peribolos_data = fetch_peribolos_file(args.org)
+    repositories = extract_repositories(peribolos_data, args.org)
+
+    if not repositories:
+        print("No repositories found in peribolos configuration")
+        sys.exit(0)
+
+    if args.repos:
+        repositories = [r for r in repositories if r in args.repos]
+        print(f"Filtering to {len(repositories)} specified repository(ies)")
+
+    # Skip org-infra itself and other excluded repos
+    excluded_repos = config.get('exclude_repos', ['org-infra'])
```

Comment:
According to above comment, 'org-infra' should always be excluded, right?  here will only exclude 'exclude_repos' if specified in sync config file.